Library/Homebrew/requirements/ruby_requirement.rb

```diff
@@ -12,7 +12,20 @@ def initialize(tags)
     which_all("ruby").detect do |ruby|
       version = /\d\.\d/.match Utils.popen_read(ruby, "--version")
       next unless version
-      Version.create(version.to_s) >= Version.create(@version)
+      version = version.to_s
+      if version.to_f > 1.9
```

Comment:
We can assume that there is at least Ruby 2.0 installed in addition to other Rubies. I'd prefer not to return that the requirement is satisfied if the Ruby ≥ 2.0 does not have `ruby.h` installed. In that case I'd rather install the brewed Ruby.